app/presenters/publishing_api_presenters/political_helper.rb

```diff
@@ -0,0 +1,12 @@
+module PublishingApiPresenters::PoliticalHelper
+private
+
+  def government
+    gov = item.government
+    {
+      title: gov.name,
+      slug: gov.slug,
+      current: gov.current?
+    }
```

Comment:
@fofr do you mean something like this?
https://github.com/alphagov/whitehall/blob/7c05142b175ae4a9586c8827e000d5a6743dbb0e/app/presenters/publishing_api_presenters/political_helper.rb#L13-L17